testsuite/python/lb.py

```diff
@@ -674,7 +674,7 @@ def test_tracers_coupling_rounding(self):
         self.system.thermostat.set_lb(LB_fluid=lbf, seed=3, gamma=self.gamma)
         rtol = self.rtol
         if lbf.single_precision:
-            rtol *= 100.
+            rtol *= 200.
```

Comment:
I had to increase this tolerance because the single-precision kernels are very sensitive to precision loss. In fact, if you regenerate the kernels enough times, you will eventually get a collide kernel where the order of operations is such that this test fails. I had to double the tolerance to avoid failures.

What is really peculiar is how with 2 MPI ranks or more, the initial LB populations are not bitwise identical in the 8 nodes sampled by the tracer. I would have expected them to be the same, since we don't use a thermalized kernel. I invested 2 days on this oddity and still don't understand the discrepancy. This oddity was not introduced by this PR, it's been here for a while.